cms/functions.php

```diff
@@ -0,0 +1,260 @@
+ * Returns all project image links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project image links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectImages(array $array):string {
+    $projectImages = '';
+    foreach ($array as $value) {
+        $projectImages .= $value['projectImage']."\n";
+    }
+    return $projectImages;
+}
+
+/* Doc Block
+ * Returns project id's from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of id's or keys in the arrays within the arrays pulled from the database.
+ */
+function returnProjectIds(array $array):string {
+    $projectIds = '';
+    foreach ($array as $value) {
+        $projectIds .= $value["id"]."\n";
+    }
+    return $projectIds;
+}
+
+/* Doc Block
+ * Returns a single project title from array content provided by database.
+ *
+ * @param $array array associative array provided by database query.
+ * @param $recId int number pointing to the record id number in the database.
+ *
+ * @return string the title content for a single record.
+ */
+function returnProjectTitle(array $array, int $recId):string { // change title to record
+    $index = $recId - 1;
+    $projectTitleArray = $array[$index];
+    $projectTitle = '';
+    $projectTitle .=$projectTitleArray['projectTitle'];
+    return $projectTitle;
+}
+
+/* Doc Block
+ * Returns project title link from array content provided by database.
+ *
+ * @param $array array associative array provided by database query.
+ * @param $recId int number pointing to the record id number in the database.
+ *
+ * @return string the title link content for a single record.
+ */
+function returnProjectTitleLink(array $array, int $recId):string {
+    $index = $recId - 1;
+    $projectTitleLinkArray = $array[$index];
+    $projectTitleLink = '';
+    $projectTitleLink .= $projectTitleLinkArray["projectTitleLink"];
+    return $projectTitleLink;
+}
+
+/* Doc Block
+ * Returns project image link from array content provided by database.
+ *
+ * @param $array array associative array provided by database query.
+ * @param $recId int number pointing to the record id number in the database.
+ *
+ * @return string the project image link for a single record.
+ */
+function returnProjectImage(array $array, int $recId):string {
+    $index = $recId - 1;
+    $projectImageArray = $array[$index];
+    $projectImage = '';
+    $projectImage .= $projectImageArray['projectImage'];
+    return $projectImage;
+}
+
+/* Doc Block
+ * Returns project id from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ * @param $recId int number pointing to the record id number in the database.
+ *
+ * @return string an id or key for a single record.
+ */
+function returnProjectId(array $array, int $recId):string {
```

Comment:
same as above

Reply:
Edited for all 4 functions that use this.